boilerplate/app/components/bullet-item/bullet-item.tsx

```diff
@@ -21,7 +21,7 @@ const BULLET: ImageStyle = {
 }
 const BULLET_TEXT: TextStyle = {
   flex: 1,
-  fontFamily: typography.primary,
+  fontFamily: typography.primary.normal,
```

Comment:
Changes to files below this one when it comes to colors and fonts are all temporary fixes and may completely break the look of the current (now legacy) components. These will all be addressed as more components are rebuilt.